health-services/project-factory/src/server/validators/campaignValidators.ts

```
@@ -577,6 +589,13 @@
     }
 }
 function validateForDupicateRows(boundaryData: any[]) {
+    boundaryData = boundaryData.map(row =>
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)